nameserver_linux.go

```diff
@@ -0,0 +1,36 @@
+package main
```

Comment:
This breaks compilation on MacOS for dev builds. Can you rename the file to `nameserver_unix.go` and add a build constraint at the head?
```
// +build darwin freebsd linux netbsd openbsd

package main
```